.travis.yml

```diff
@@ -19,37 +19,37 @@ before_install:
 install: travis_retry composer update --prefer-dist
 
 script:
-  - if [[ "$DB" == "mysql" || "$DB" == "mariadb" ]]; then mysql -e "CREATE SCHEMA doctrine_tests; GRANT ALL PRIVILEGES ON doctrine_tests.* to travis@'%'"; fi
+#  - if [[ "$DB" == "mysql" || "$DB" == "mariadb" ]]; then mysql -e "CREATE SCHEMA doctrine_tests; GRANT ALL PRIVILEGES ON doctrine_tests.* to travis@'%'"; fi
   - ENABLE_SECOND_LEVEL_CACHE=0 ./vendor/bin/phpunit -v -c tests/travis/$DB.travis.xml
-  - ENABLE_SECOND_LEVEL_CACHE=1 ./vendor/bin/phpunit -v -c tests/travis/$DB.travis.xml --exclude-group performance,non-cacheable,locking_functional
+#  - ENABLE_SECOND_LEVEL_CACHE=1 ./vendor/bin/phpunit -v -c tests/travis/$DB.travis.xml --exclude-group performance,non-cacheable,locking_functional
 
 jobs:
   include:
-    - stage: Test
-      env: DB=mariadb
-      addons:
-        mariadb: 10.1
-
-    - stage: Test
-      env: DB=mysql MYSQL_VERSION=5.7
-      php: 7.1
-      before_script:
-        - ./tests/travis/install-mysql-$MYSQL_VERSION.sh
-      sudo: required
-
-    - stage: Test
-      env: DB=mysql MYSQL_VERSION=5.7
-      php: 7.2
-      before_script:
-        - ./tests/travis/install-mysql-$MYSQL_VERSION.sh
-      sudo: required
-
-    - stage: Test
-      env: DB=mysql MYSQL_VERSION=5.7
-      php: nightly
-      before_script:
-        - ./tests/travis/install-mysql-$MYSQL_VERSION.sh
-      sudo: required
+#    - stage: Test
+#      env: DB=mariadb
+#      addons:
+#        mariadb: 10.1
+#
+#    - stage: Test
+#      env: DB=mysql MYSQL_VERSION=5.7
+#      php: 7.1
```

Comment:
Didn't we remove PHP 7.1 from `develop`?

Reply:
Yes, but @guilhermeblanco lost that commit during recent rebase. We should re-apply it.

Reply:
😭